app/assets/javascripts/admin/addon/controllers/admin-users-list-show.js

```diff
@@ -106,4 +120,79 @@ export default class AdminUsersListShowController extends Controller.extend(
       asc,
     });
   }
+
+  @action
+  toggleBulkSelect() {
+    this.bulkSelect = !this.bulkSelect;
+    this.displayBulkActions = false;
+    this.bulkSelectedUsersMap = {};
+    this.bulkSelectedUserIdsSet = new Set();
+  }
+
+  @action
+  bulkSelectItemToggle(userId, event) {
+    if (event.target.checked) {
+      if (this.bulkSelectedUserIdsSet.size === MAX_BULK_SELECT_LIMIT) {
```

Comment:
I think this needs to be `>=` not `===`, otherwise you can bypass it with bulk selection